.changeset/lovely-hotels-bake.md

```diff
@@ -0,0 +1,5 @@
+---
+"@thirdweb-dev/react": patch
```

Comment:
ur also changing things in sdk so missing a changeset for that

Reply:
done